pinax/templates/templates/pinax/admin/pinax/points/awardedpointvalue/change_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "admin/change_list.html" %}
```

Comment:
I think this is at the wrong path.

Since this is an admin override I think you want it at the path:

```
pinax/templates/templates/admin/pinax/points/awardedpointvalue/change_list.html
```

but not entirely sure.  It might be:

```
pinax/templates/templates/admin/pinax_points/awardedpointvalue/change_list.html
```

Because of the app lable: https://github.com/pinax/pinax-points/blob/master/pinax/points/apps.py#L8

Reply:
Good catch - those long paths got me 😆

Reply:
This admin file looks out of place to not be in the points directory. Are you sure it should be in its own separate admin folder in pinax-templates? If pinax-points were included in a starter project, perhaps the admin templates would be in the project templates folder. https://github.com/jaydenwindle/pinax-templates/tree/move_pinax_points_templates/pinax/templates/templates/pinax